### ISSUE TITLE
Tooltips not accessible with keyboard

### ISSUE THREAD

**Issue by boemedia** [woocommerce/woocommerce#29081]
**Prerequisites (mark completed items with an [x]):**
- [X] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [X] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

**Issue Description:**

WooCommerce uses tooltips to provide additional information in WP-admin on certain menu items. However, this information is only accessible if you can use a mouse. When you tab through the settings of the product in WP-admin, the tooltip is skipped. Same when you use voice over.

![image](https://user-images.githubusercontent.com/29769622/107493648-24b06480-6b8e-11eb-990c-392f6f82f536.png)

**Issue Solution**
Obviously, making the tooltips accessible. But as a first step, it would be nice if there would be a place where developers can add a descriptive text to the label to give a short cue of what needs to be done. The tooltip should only be used for longer , additional instructions. Information that is essential for users to perform a task and doesn't require much space, should be visible any time.

![image](https://user-images.githubusercontent.com/29769622/107494035-98527180-6b8e-11eb-8e7f-383742f5aee4.png)

The tooltip text in the screenshot here is very short and might as well be visible in the UI all the time. In this case, the information is not that relevant, but solely posted as an example.

**Comment by rrennick** [comment]
Addressed in https://github.com/woocommerce/woocommerce/pull/37808.
